lib/commands/europeana/portal/docker/docker_command.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'dotenv'
+
+module Europeana
+  module Portal
+    class DockerCommand < Thor::Group
+      include Thor::Actions
+      include Rails::Generators::Actions
+
+      source_root File.expand_path('templates', __dir__)
+
+      argument :rails_env, type: :string, default: 'development', desc: 'Rails env'
+
+      class_option :europeana_api_key, type: :string, desc: 'Europeana API key'
+      class_option :redis_db_index, type: :numeric, default: 0, desc: 'Redis database index'
+
+      def copy_docker_compose
+        template 'docker-compose.yml', 'docker-compose.yml'
+      end
+
+      def copy_docker_env
+        template '.env.docker', '.env.docker'
+      end
+
+      def copy_minio_policy
+        template 'minio-s3-policy.json', minio_policy_path
+      end
+
+      def copy_rails_env
+        template '.env.rails', ".env.#{rails_env}"
```

Comment:
could we append the values rather than overwriting the entire file? That way anything already specified in the .env file won't be lost, but since the last assignment will take precedence the relevant settings will still be applied.